### PR TITLE
Add books section into the QGIS Website.

### PR DESCRIPTION
**Pull request by yjacolin**

### CODE REVIEW

**Comment by g-sherman** [comment]
There's a couple at locatepress.com :)

**Comment by yjacolin** [comment]
@g-sherman I will look into the website, thanks!

**Comment by yjacolin** [comment]
@g-sherman I added locatepress books. Could you check that none are missing?

**Comment by g-sherman** [comment]
Looks fine for locatepress books...

File: source/site/forusers/books/index.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Mastering QGIS
+***************
+
+Dive into QGIS and master geospatial data with this advanced guide, created to 
+help you develop maps and applications that let you explore the world in new 
+ways.
+
+* https://www.packtpub.com/application-development/mastering-qgis
+* Author(s): Kurt Menke, GISP et al.
```
Comment (luipir) [inline source/site/forusers/books/index.rst]: Authors, are ;):
Kurt Menke, GISP - Dr. Richard Smith Jr., GISP - Dr. Luigi Pirelli - Dr. John Van Hoesen, GISP

File: source/site/forusers/books/index.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+* Date: March 2015
+
+QGIS Blueprints
+****************
+
+Develop analytical location-based web applications with QGIS 
+
+* https://www.packtpub.com/hardware-and-creative/qgis-blueprints
+* Author(s): Ben Mearn
+* Date: Sseptember 2015
```
Comment (luipir) [inline source/site/forusers/books/index.rst]: s/Sseptember/September/

**Comment by yjacolin** [comment]
@rduivenvoorde, could you merge my PR?

**Comment by rduivenvoorde** [comment]
@yjacolin yep, will have a look this evening, cannot check/build here...

**Comment by rduivenvoorde** [comment]
@yjacolin thanks Yves, sorry for taking so long to pull

What about adding some images?

**Comment by yjacolin** [comment]
@rduivenvoorde I agree that some image could improve the page. but I wonder about legal issue. Currently, I copy-pasted the description from website. do you think this is a real issue?

I could try to improve the page, but probably not the next week as I am traveling.

**Pull request merged**